d2l-search-results-count.html

```diff
@@ -0,0 +1,72 @@
+<link rel="import" href="../polymer/polymer-element.html">
+<link rel="import" href="d2l-search-results-count-localize-behavior.html">
+
+<dom-module id="d2l-search-results-count">
+	<template strip-whitespace>
+		<style>
```

Comment:
You may need to add an empty rule due to linting issues ([example](https://github.com/BrightspaceUI/menu/blob/master/d2l-menu-item-radio.html#L16))